velox/exec/tests/TableWriteTest.cpp

```diff
@@ -2760,11 +2760,14 @@ TEST_P(AllTableWriterTest, tableWriterStats) {
   // bucketNum, bucket number is 4
   const int numWrittenFiles =
       bucketProperty_ == nullptr ? numBatches : numBatches * 4;
+  const int32_t ORC_HEADER_LEN{3};
```

Comment:
Can we add comment here to explain (the description from the issue: https://github.com/facebookincubator/velox/issues/7169)